Automattic/newspack-newsletters
# fix: use hex value for group background colors in default layouts

### All Submissions:






### Changes proposed in this Pull Request:

Fixes an additional issue with Group Block background color. Three of the four default layouts use `very-light-gray` as their background. This color option is no longer available, so the blocks have no background color when rendered to email markup. This PR replaces them with the hex color `#f0f0f0`.

### How to test the changes in this Pull Request:

1. On `master`, create Newsletters with the "Breaking News," "Daily/Weekly (no image)," or "Daily/Weekly" default layouts.
2. Send a test email, observe the Group block backgrounds are not present in the email.
3. Switch to this branch and repeat.
4. Observe the backgrounds are present in the email.

### Other information:
